src/components/diff/PlainTextDiff.ts

```diff
@@ -70,7 +96,11 @@ export class PlainTextDiff extends Widget implements Git.Diff.IDiffWidget {
     this.ready
       .then(() => {
         if (this._challenger !== null && this._reference !== null) {
-          this.createDiffView(this._challenger, this._reference);
+          this.createDiffView(
+            this._challenger,
+            this._reference,
+            this._isConflict ? this._base : undefined
```

Comment:
A good practice is to never set `undefined` to distinguish between out of bound value and explicitly set empty value.

```suggestion
            this._isConflict ? this._base : null
```